packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -63,7 +63,7 @@ const ProfileImage = (props) => {
   /* eslint-enable react/forbid-dom-props */
 
   if (props.src) {
-    return (<div><TerraImage placeholder={placeholderImage} {...props} /></div>);
+    return (<div className={cx('image-container')}><TerraImage placeholder={placeholderImage} {...props} /></div>);
```

Comment:
what happens if we remove the div altogether?

Reply:
That also fixes the issue. Updated it here https://github.com/cerner/terra-core/pull/3422/commits/5e756c9c6a21ba93d3bc8ca99d0590da5e3cb10e.